src/wp-includes/block-bindings/sources/post-meta.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Add the post_meta source to the Block Bindings API.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function post_meta_source_callback( $source_attrs ) {
+	if ( ! isset( $source_attrs['key'] ) ) {
+		return null;
+	}
+
+	// Use the postId attribute if available
+	if ( isset( $source_attrs['postId'] ) ) {
+		$post_id = $source_attrs['postId'];
```

Comment:
Do we have use-cases where the post id is an attribute and not something in the context?

Reply:
I think it should use at least two params, the second one is `$block_instance` where we should read context as noted in the code and GitHub comments. The third parameter is the name of the attribute that gets updated. I was on the fence with that one, as only Pattern Overrides need it today when processing the block context.

Reply:
@SantosGuillamot might know better. I would assume we rather read it from the context or other block attributes through `$block_instance` param.

Reply:
The idea was to allow people to select a specific ID to connect to. So, instead of connecting to the post title of the context, you connect to a specific post title.

I added it more as an example of other attributes that could make sense at some point.